README.md

```diff
@@ -13,6 +13,7 @@ System integration is the process of linking together different IT systems (co
   - [BRE](#bre)
   - [BPM](#bpm)
   - [Data Mapping Solution](#data-mapping-solution) 
+  - [eiPaaS](#eipaas)
```

Comment:
Embedded iPaaS is better as eiPaaS means enterprise integration platform as a service